src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball []balls = new Ball[3];
+        for (Ball ball:balls) {
+            ball = lottery.getRandomBall();
+            System.out.println(ball.toString());
+        }
```

Comment:
Do not create unnecessary variables
All magic numbers, strings, symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < NUMBER_LOTTERY; i++) {
            System.out.println(lottery.getRandomBall());
        }
```